src/actions/walletActions.js

```diff
@@ -1,19 +1,27 @@
 import Web3 from 'web3'
 import LedgerNanoS from '../ledgerSigner'
 import utils from '../utils'
-import BN from 'bn.js'
 import { goToStep } from './navigationActions'
 import { Base64 } from 'js-base64'
 import { getTransferData } from '../drive.js'
 import ERC20 from '../ERC20'
 
 const ledgerNanoS = new LedgerNanoS()
 
+function syncLedgerAccountInfo (cryptoType, accountIndex = 0) {
+  return (dispatch, getState) => {
```

Comment:
what are (dispatch, getState) doing here?

Reply:
Forgot to remove. fixed.